components/org.wso2.carbon.identity.application.authenticator.adapter/src/main/java/org/wso2/carbon/identity/application/authenticator/adapter/util/AuthenticatedUserBuilder.java

```diff
@@ -174,6 +176,9 @@ private Map<ClaimMapping, String> resolveUserNameAndClaimsFromResponse() {
 
         Map<ClaimMapping, String> userAttributes = new HashMap<>();
         for (AuthenticatedUserData.Claim claim : userFromResponse.getUser().getClaims()) {
+            if (GROUP_CLAIM.equals(claim.getUri())) {
+                continue;
```

Comment:
Better add  a diagnostic to denote group claim is ignored and mention the expected way